src/encdatavault_fmt_plug.c

```diff
@@ -276,82 +339,118 @@ static int crypt_all(int *pcount, struct db_salt *salt)
 		any_cracked = 0;
 	}
 
-	nb_keys = 1 << (cur_salt->algo_id - 1);
+	nb_keys = 1 << (enc_cur_salt->algo_id - 1);
 
 #ifdef _OPENMP
-#pragma omp parallel for
+	#pragma omp parallel for
```

Comment:
We normally don't indent `#pragma`, nor contents of an `#ifdef` block.

Reply:
`#pragma` shouldn't be indented - please fix.

Reply:
OK, I'll give up on this one. ;-)

Reply:
Sorry I was testing indent, it redo it.